fluidBuild.config.cjs

```diff
@@ -47,6 +47,7 @@ module.exports = {
 				"eslint",
 				"good-fences",
 				"depcruise",
+				"check:exports",
```

Comment:
You may also want to add a task definition for this, something like `"check:exports": []`. Technically that is only needed if the task is script:false, but we generally define them. Not a huge deal and happy to talk about changing the "pattern" here and only defining tasks that must be explicitly defined.

Reply:
It should have been there. And it should depend on "api" since those are the things it checks.